cudaaligner/benchmarks/CMakeLists.txt

```diff
@@ -8,9 +8,19 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-# Add benchmarks
-add_subdirectory(singlealignment)
-add_subdirectory(singlebatchalignment)
+
+project(benchmark_cudaaligner_singlebatch)
```

Comment:
Let's call this `benchmark_cudaaligner` like we did for `cudapoa`. READMEs would need to be updated as well.